src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -205,6 +207,20 @@ def _get_grades() -> None:
     )
 
 
+@catch_exceptions
```

Comment:
Just FYI, this will be something to move to the shared library.  Will be very easy to do.

Reply:
Sounds good